addons/zebra-striping/addon.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Alternating nested block colors",
+  "description": "Makes blocks of the same category alternate between lighter and darker shades when nested inside eachother.",
```

Comment:
"Zebra stripes" should be mentioned somewhere in the addon name or description because people will probably search for that.

Reply:
@mxmou I think it already finds it at searching for "zebra stripes", because of the addon ID. 
(I'm not sure though that you meant this.)
![image](https://user-images.githubusercontent.com/101597941/184086891-087c6fe5-6335-4859-9266-9476baba0038.png)

Reply:
The addon ID is a technical detail. Yes, it's possible to search by ID, but that's mainly intended for developers.

Reply:
> Yes, it's possible to search by ID,...

Is it just possible or the only option as well? Because I think the search engine always look at the addon ID too in any search cases.

Reply:
It does but I don't think it should have to just to show relevant results.

Reply:
> It does but I don't think it should have to just to show relevant results.

Why do you think that? I don't see any issues (mostly meaning confusions) it could cause.

Reply:
Sorry for debating, I'm just curious about your reasons.

Reply:
Addon IDs are not translated and (in the stable version) never shown to the user.

Reply:
Oh ok, you are totally right. But I don't think that someone will search for zebra-stripes in another language because it's not translated anywhere. So in this very case it doesn't matter that much, as I think, but in general it would be logical.